### PR TITLE
Ensure that the Java samples build.

### PR DESCRIPTION
**Pull request by tswast**
This requires a few fixes to `pom.xml` files.  Adds a simple Travis
config to ensure the Java samples continue to build.

### CODE REVIEW

File: java/simple-performance-test/pom.xml [hunk 1]
```diff
@@ -97,15 +95,6 @@
     </dependency>
   </dependencies>
 
-  <repositories>
```
Comment (sduskis) [inline java/simple-performance-test/pom.xml]: Please add the snapshot repositories back in.

File: travis.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  echo "
+Usage: $0 { java_jdk8 |
+            java_oracle8 }
+"
+  exit 1
+fi
+
+set -e  # exit immediately on error
+set -x  # display all commands
+eval "build_$1"
```
Comment (mbrukman) [inline travis.sh]: Why use `eval` when `case` will do? Using `case` will also get you validation of input arg, which is not done here with `eval`. Can this be combined with the contents of `use_java()` above?

File: java/simple-performance-test/pom.xml (hunk 1 above)
Comment (tswast) [inline java/simple-performance-test/pom.xml]: This is a duplicate definition. The sample does not compile with this here.

The snapshots repository is defined above. https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/pull/154/files#diff-6de1b0be654f8f9c45ef8657c367c531R35

File: travis.sh (hunk 2 above)
Comment (tswast) [inline travis.sh]: I'm going to add lines for Scala and Python, as well, so I will not be combining with `use_java()`. I would have just used the `language: java` Travis image if there were just Java samples.

As for `eval` versus `case` wouldn't I end up having to duplicate all the different definitions here in order to do that? I've modeled this after https://github.com/google/protobuf/blob/master/tests.sh#L344 which, I'd like to stay consistent with.

**Comment by tswast** [comment]
@lesv PTAL.

**Comment by lesv** [comment]
LGTM - for Phase 1

**Pull request merged**